python/tvm/relay/op/vision/multibox.py

```diff
@@ -0,0 +1,38 @@
+"""Multibox operations."""
+from __future__ import absolute_import as _abs
+from . import _make
+
+def multibox_prior(data,
+                   sizes,
+                   ratios,
+                   steps,
+                   offsets,
+                   clip):
+    """Generate prior(anchor) boxes from data, sizes and ratios.
+
+    Parameters
+    ----------
+    data : relay.Expr
+        The input data tensor.
+
+    sizes : tuple of float
+        Tuple of sizes for anchor boxes.
+
+    ratios : tuple of float
+        Tuple of ratios for anchor boxes.
+
+    steps : Tuple of float
+        Priorbox step across y and x, -1 for auto calculation.
+
+    offsets : tuple of int
+        Priorbox center offsets, y and x respectively.
+
+    clip : boolean
+        Whether to clip out-of-boundary boxes.
+
```

Comment:
`MultiBoxPriorAttr` above mention default values for many of the arguments. Could you ensure that this comment matches those?